### PR TITLE
RC-23654 Changes for syncing excluded namespaces.

### PR DESCRIPTION
**Pull request by rafay-vivek**

### CODE REVIEW

File: examples/resources/rafay_project/resource.tf [hunk 1]
```diff
@@ -17,6 +17,7 @@ resource "rafay_project" "tfdemoproject2" {
   }
   spec {
     default = false
+    sync_excluded_namespaces = ["one", "two"]
```
Comment (krishnarafay) [inline examples/resources/rafay_project/resource.tf:20]: instead of `one,two` can rename to `ns1,ns2,ns3` something like this..

Comment (krishnarafay) [reply examples/resources/rafay_project/resource.tf:20]: can you update https://github.com/RafaySystems/terraform-provider-rafay/blob/master/docs/resources/project.md doc also..?

File: rafay/resource_project.go [hunk 2]
```diff
@@ -279,6 +279,10 @@ func expandProjectSpec(p []interface{}) (*systempb.ProjectSpec, error) {
 		obj.DriftWebhook = expandProjectDriftWebhook(v)
 	}
 
+	if v, ok := in["sync_excluded_namespaces"].([]interface{}); ok {
```
Comment (krishnarafay) [inline rafay/resource_project.go:282]: can you update https://github.com/RafaySystems/terraform-provider-rafay/blob/master/docs/resources/project.md doc also..?

File: examples/resources/rafay_project/resource.tf (hunk 1 above)
Comment (rafay-vivek) [reply examples/resources/rafay_project/resource.tf:20]: okay, will make that change.

File: rafay/resource_project.go (hunk 2 above)
Comment (rafay-vivek) [reply rafay/resource_project.go:282]: okay, will make that change.

**Review by krishnarafay — APPROVED** [review]
looks good.. please update `go.mod` wrt `rafay-common`.

**Review by nikhil-rafay — APPROVED** [review]

**Review by janardhan-rafay — APPROVED** [review]

**Pull request merged**